docs/index.md

```diff
@@ -0,0 +1,116 @@
+        
+        def type = "java"
+        def product = "labs"
+        def component = "mohanalatest"
+        
+        withPipeline(type, product, component) {
+        }
+3. Create Helm values file with following contents.
+
+   CFT: values.sandbox.template.yaml
+
+   SDS: values.sbox.template.yaml
+   
+    ```
+        java:
+          # Don't modify below here
+          image: ${IMAGE_NAME}
+          ingressHost: ${SERVICE_FQDN}
+4. Get the Pull Request reviewed and merged.
+
+5. Login to Jenkins and select "HMCTS-Labs" folder. Scan the organization by clicking on `Scan Organization Now`. New repository should be listed under repositires after scan finishes. Logs can be moinitored under `Scan Organization Log`. Any github repos that starts with `labs*` will be listed as part of this scan.
+
+   CFT: [jenkins](https://sandbox-build.platform.hmcts.net)
+
+   SDS: [jenkins](https://sds-sandbox-build.platform.hmcts.net)
+
+6. Run the jenkins pipeline against the `master` branch.
+
+7. We load balance across AKS clusters using `Azure Application Gateway`. Add couple of lines of config for the application.
+
+   CFT:  [config file](https://github.com/hmcts/azure-platform-terraform/blob/master/environments/sbox/backend_lb_config.yaml)
+
+   SDS:  [config file](https://github.com/hmcts/sharedservices-azure-platform/blob/master/environments/sbox/backend_lb_config.yaml)
+    ```
+           #labs
+               - product: "labs"
+                 component:     #githubreponame without "labs" prefix
+
+8. Azure Front Door is our entry point in HMCTS. Add following line of code for the app.
+
+   CFT: [config file](https://github.com/hmcts/azure-platform-terraform/blob/master/environments/sbox/sbox.tfvars)
+    ```
+     {
+        product          = "labs"
+        name             = "#githubreponame without "labs" prefix"
+        custom_domain    = "your-app.service.core-compute-sandbox.internal"
+        backend_domain   = ["firewall-sbox-int-palo-sbox.uksouth.cloudapp.azure.com"]
+        certificate_name = "wildcard-sandbox-platform-hmcts-net"
+      }
+    ```
+   
+   SDS: [config file](https://github.com/hmcts/sharedservices-azure-platform/blob/master/environments/sbox/sbox.tfvars)
+
+    ```
+       {
+        product          = "labs"
+        name             = "#githubreponame without "labs" prefix"
+        custom_domain    = "your-app.sandbox.platform.hmcts.net"
+        backend_domain   = ["firewall-sbox-int-palo-sdsapimgmt.uksouth.cloudapp.azure.com"]
+        certificate_name = "wildcard-sandbox-platform-hmcts-net"
+      }
+
+
+9. We practise [GitOps](https://www.weave.works/technologies/gitops/) for application deployment to Kubernetes.
+
+   CFT: [repo](https://github.com/hmcts/cnp-flux-config)
+
+   SDS: [repo](https://github.com/hmcts/shared-services-flux)
+
+10. Access the deployed application using the URL mentioned in FrontDoor code.
+    ```
+    custom_domain    = "your-app.sandbox.platform.hmcts.net"   
```

Comment:
the app will be accessed on the .internal domain for CFT and provide the full url with http in it so that people don't try add https

also the code block here is broken (but should be removed anyway)